packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -134,4 +138,21 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
 
     return this.wrapResponse(res)
   }
+
+  async simulate(address, input): Promise<Number> {
+    const msg = new MsgExecuteContract(this.wallet.key.accAddress, address, input)
+
+    const signerData: SignerData = {
+      sequenceNumber: await this.wallet.sequence(),
+      publicKey: this.wallet.key.publicKey,
```

Comment:
We'll need a way to get the `signer` sequence number
The public key should be the signer